### ISSUE TITLE
Missing customer headers for ResourceAction

### ISSUE THREAD

**Issue by Mduchamp** [troyanskiy/rest-core#31]
I'm trying to submit custom headers to an http request. I'm using the following code:

```
@ResourceAction( {
        headers: { 'myHeader' : 'myHeaderValue' },
        method : ResourceRequestMethod.Post,
        path : '/my/path',
    } )
    myMethod : IResourceMethod< { myString : string }, void >;
```

As far as I am able to tell, this should be working, but I never see the customer headers when I look at the actual request. I see the method and path headers fine, as well as the payload. But my custom headers are just missing. Is this a bug? Or am I doing something wrong?

Thanks!

**Comment by troyanskiy** [comment]
Hello.
Looks like a bug.
Will check the issue tomorrow.

Thanks!

**Comment by kevwiches** [comment]
@troyanskiy any update on this one?

**Comment by troyanskiy** [comment]
Hello @all
I did some refactoring and repository restructuring.
From now on new repository is located here https://github.com/troyanskiy/ngx-resource
Please create new issues in new repo.
I will close all the issues in that repo.

Also I have released 7.0.0-beta.5 version with `Observables` support. Please test it and create new issues in new repo if you will find some bugs.

To update to new version run
`npm i --save @ngx-resource/core@beta @ngx-resource/handler-ngx-http@beta`

### Breaking changes:
#### `ResourceGlobalConfig`
* removed `asPromise` and `asResourceResponse`
* added `returnAs` with enum type `ResourceActionReturnType`; Default is `ResourceActionReturnType.Observable`

#### `IResourceParams` and `IResourceAction`
* removed `asPromise` and `asResourceResponse`
* added `returnAs` with enum type `ResourceActionReturnType`; Default is `ResourceActionReturnType.Observable`

### Deprecation
* `IResourceMethod` -> `IResourceMethodPromise`
* `IResourceMethodStrict` -> `IResourceMethodPromiseStrict`
* `IResourceMethodFull` -> `IResourceMethodPromiseFull`
* `IResourceMethodStrictFull` -> `IResourceMethodPromiseStrictFull`
* `ResourceCRUD` -> `ResourceCRUDPromise`

### New
* `ResourceCRUDObservable` - same as ResourceCRUDPromise, but typed to Observable

I still have to update Docs and provide migration guide.

Thank you @all !!!
